logging/tests/unit/handlers/transports/test_background_thread.py

```diff
@@ -278,14 +281,15 @@ def join(self, timeout=None):
 
 
 class _Batch(object):
+    from google.cloud.logging.logger import _GLOBAL_RESOURCE
 
     def __init__(self):
         self.entries = []
         self.commit_called = False
         self.commit_count = None
 
-    def log_struct(self, info, severity=logging.INFO):
-        self.log_struct_called_with = (info, severity)
+    def log_struct(self, info, severity=logging.INFO, resource=_GLOBAL_RESOURCE):
+        self.log_struct_called_with = (info, severity, resource)
```

Comment:
Instead, make `resource=None` in the constructor and then change this to the following:

```python
from google.cloud.logging.logger import _GLOBAL_RESOURCE

if resource is None:
    resource = _GLOBAL_RESOURCE

self.log_struct_called_with = (info, severity, resource)
```